.github/workflows/build-ios.yml

```diff
@@ -11,6 +11,9 @@ jobs:
     runs-on: macos-latest
     permissions:
       contents: write
+
```

Comment:
can you change on push to on release? Or do we need the build to run whenever we push something and not want to release?

Reply:
Also doesn't it make more sense if instead of creating a release, it attaches the files to latest release? (I mean the other way around, instead of whenever we push something, it creates a release, whenever we release, it attaches the files to it)